content/02-Platform/Frontend/forms/tutorial.md

```diff
@@ -0,0 +1,232 @@
+
+<img src="/img/docs/forms/first-field.png"/>
+
+That's not the most exciting field, but it's a field! We can add a title to it by adding to `uiSchema`:
+
+```js
+page1: {
+  path: 'first-page',
+  title: 'First Page',
+  uiSchema: {
+    myField: {
+      'ui:title': 'My field label'
+    }
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      myField: {
+        type: 'string'
+      }
+    }
+  }
+}
+```
+
+That makes it look a little more presentable:
+
+<img src="/img/docs/forms/field-with-label.png"/>
+
+Note that `uiSchema` doesn't follow exactly the same structure as `schema`: you don't need the `properties` object. This is because `uiSchema` treats everything without a `ui:` prefix as a field name, which one exception for array fields.
+
+Changing the the `type` property in your field will change the data accepted and also the way it displays on the form. You can change it to be `boolean` and get a checkbox and `number` to get a number input. If you want a `select` box, you use JSON Schema's `enum` property:
+
+```js
+page1: {
+  path: 'first-page',
+  title: 'First Page',
+  uiSchema: {
+    myField: {
+      'ui:title': 'My field label'
+    }
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      myField: {
+        type: 'string',
+        'enum': [
+          'First option',
+          'Second option'
+        ]
+      }
+    }
+  }
+}
+```
+
+That will get you a select box with options:
+
+<img src="/img/docs/forms/select-field.png"/>
+
+Some types of data might have different valid ways of asking the user for input. For example, a field that uses `enum` could also use radio buttons. You can change that with `ui:widget`:
+
+```js
+page1: {
+  path: 'first-page',
+  title: 'First Page',
+  uiSchema: {
+    myField: {
+      'ui:title': 'My field label',
+      'ui:widget': 'radio'
+    }
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      myField: {
+        type: 'string',
+        'enum': [
+          'First option',
+          'Second option'
+        ]
+      }
+    }
+  }
+}
+```
+
+Now the form offers two radio buttons to choose from:
+
+<img src="/img/docs/forms/radio-buttons.png"/>
+
+You can also mark fields as required, which will prevent you from moving to the next page without filling them out:
+
+```js
+page1: {
+  path: 'first-page',
+  title: 'First Page',
+  uiSchema: {
+    myField: {
+      'ui:title': 'My field label',
+      'ui:widget': 'radio'
+    }
+  },
+  schema: {
+    type: 'object',
+    required: ['myField'],
+    properties: {
+      myField: {
+        type: 'string',
+        'enum': [
+          'First option',
+          'Second option'
+        ]
+      }
+    }
+  }
+}
+```
+
+If you do fill in the required information and click Continue, you'll end up on the review page. The review page for our forms is generated based on your chapters and pages and provides a quick way to review the data that you've entered:
```

Comment:
Would it be worth adding something like `Note that if you refresh in the middle of the form, your data will be lost and the review page won't have any content to edit` here?